rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+tab_spaces = 2
```

Comment:
Hmm, I prefer keeping the tab_spaces to the default 4.

Reply:
4 is the default style so let's stick with that, (all this is just to learn to use github anyways).  Done!